content/docs/command-reference/version.md

```diff
@@ -36,36 +34,33 @@ usage: dvc version [-h] [-q | -v]
 
 The detail of DVC version depends upon the way of installing DVC.
 
-- **Official release**: [These instructions](/doc/install) include ways to
-  install DVC using the official package stored in Python Packaging Authority.
-  We mark these official releases with tags on DVC's repository. Any issues
-  reported with the official build can be traced using the `_BASE_VERSION`
-  constant
-  [in our core repo](https://github.com/iterative/dvc/blob/master/dvc/version.py).
-  For example `0.40.2`.
+- **Official release**: a stable version of DVC (for example `1.7.4`), that can
+  be [installed](/doc/install) using the official package stored in
+  [Python Packaging Authority](https://www.pypa.io/) or with a binary. These
+  releases are tagged in the DVC repository (see
+  [changelog](https://github.com/iterative/dvc/releases) for release history).
 
-  > 💡 To only get this number, use `dvc -V`.
+  If there are any issues reported with the official build, they can be traced
+  using the official version number.
 
-- **Development version**: `pip install git+git://github.com/iterative/dvc` will
-  install DVC using the `master` branch of DVC's repository. Another way of
-  setting up the development version is to clone the repository and run
-  `pip install .`.
+- **Development version**: is a pre-release version of DVC (for example
+  `1.7.4+292cab`). It can be
+  [installed](/doc/user-guide/contributing/core#development-environment) using
+  the master branch of DVC's repository. This version contains all the changes
+  done in the master branch after the latest official release.
 
-  The master branch is constantly updated with changes that might not be ready
-  to publish yet. Therefore, installing with this method might result in usage
-  issues. To trace any error reported with this setup, we need to know exactly
-  which version is being used. For this we rely on a Git commit hash that is
-  displayed in the output of `dvc version` like this: `0.40.2+292cab.mod`. The
-  part before `+` is the `_BASE_VERSION`, and the following part is the commit
-  hash of the tip of the `master` branch. The optional suffix `.mod` means that
-  code is modified.
+  Installing development version may result in usage issues, as it contains
+  changes that might not be ready to publish yet. Any error reported with this
+  setup can be traced using Git commit hash displayed in the output of
+  `dvc version` like this: `1.7.4+292cab.mod`. The part before `+` is the
```

Comment:
> `1.7.4+292cab.mod`

A different example was given in ther previous paragaph: `1.7.4+292cab`. What's the difference? It's confusing to have 2.

Reply:
Ah yeah. "The optional suffix `.mod` means that code is modified."

Hm... Still I would only keep this 2nd dev version number example.